web/styles/app_variables.css

```diff
@@ -747,6 +747,7 @@
     /* Settings table colors */
     --color-border-table-striped: hsl(0deg 0% 0% / 20%);
     --color-border-table-bordered: hsl(0deg 0% 0% / 20%);
+    --color-background-notification-table: hsl(0deg 0% 0% / 20%);
```

Comment:
I think we can rename this to make it clear that this color is only used in table header.

Reply:
Updated